instance/models/mixins/openedx_config.py

```diff
@@ -311,3 +314,50 @@ def _get_configuration_variables(self):
             })
 
         return template
+
+    def _get_prometheus_variables(self):
+        """Get all Prometheus-related ansible variables specific to Open edX appservers."""
+        return {
+            **super()._get_prometheus_variables(),
+            'NODE_EXPORTER_NGINX_CONFIG_PATH': '/edx/app/nginx/sites-available/node-exporter',
+            'NODE_EXPORTER_HTPASSWD_PATH': '/edx/app/nginx/nginx.htpasswd',
+            'NODE_EXPORTER_USE_SSL': False,
+        }
+
+    def _get_consul_variables(self):
+        """Get all Consul-related ansible variables specific to Open edX appservers."""
+        return {
+            **super()._get_consul_variables(),
+            'consul_service_config': {
+                'service': [
+                    {
+                        "id": "edxapp-lms",
+                        "name": "edxapp",
+                        "port": 8000,
+                        "tags": ["edxapp", "lms"],
+                        "enable_tag_override": True,
+                        "checks": [
```

Comment:
It's fine to remove the checks for now – using the heartbeat URL from the load balancer should be good enough.

Reply:
:+1: makes it easier!